mopro-msm/build.rs

```diff
@@ -25,16 +25,12 @@ fn compile_shaders() {
             get_sdk(),
             "metal",
             "-c",
-            "-frecord-sources",
             shader_path.to_str().unwrap(),
             "-o",
             air_output.to_str().unwrap(),
         ];
 
-        if std::env::var("PROFILE")
-            .map(|profile| profile == "release")
-            .unwrap_or(false)
-        {
+        if cfg!(feature = "profiling-release") {
             args.push("-frecord-sources");
```

Comment:
Some questions here: 
1. Do we generate `Metal Library Symbol File` via this argument (`-frecord-release`)?
2. do we use the `symbol files` to do the profile our shaders?

Reply:
Yeah, sure

1. We do, according to [this](https://developer.apple.com/documentation/metal/shader_libraries/metal_libraries/generating_and_loading_a_metal_library_symbol_file) `-frecord-release` this attribute does exactly this, that's why I moved it into the separate feature.
2. It's not quite, we're not required these symbols to be provided to exactly be capable to profile, but we still required these symbols to be provided to be capable to see what exactly shader is running. They're pretty much the same as the symbol files for ordinary binaries, they do nothing more but to enrich debug data with the exact function names (shader names in case of metal shaders obviously).
3. According the same source we can skip the `.air` intermediate compilation step and to compile shaders to its binary representation in a single shot, but I decided to implement this improvement after the #13 will be merged.